spec/index.html

```diff
@@ -1002,6 +1039,7 @@ <h6 class="defcap">RDF Axioms</h6>
             rdf:nil       rdf:type rdf:List .
             rdf:_1        rdf:type rdf:Property .
             rdf:_2        rdf:type rdf:Property .
+            <span style="background-color: Cyan;color: Black">rdf:reifies   rdf:type rdf:ReificationProperty .</span>
             ...
```

Comment:
```suggestion
            <span style="background-color: Cyan;color: Black">rdf:reifies   rdf:type rdf:ReificationProperty .</span>
            rdf:_1        rdf:type rdf:Property .
            rdf:_2        rdf:type rdf:Property .
            ...
```
Move new line up so the `...` remain directly after `rdf:_2`.